api/proto/teleport/legacy/types/types.proto

```diff
@@ -2492,6 +2492,15 @@ message AccessCapabilitiesRequest {
     (gogoproto.jsontag) = "resource_ids,omitempty",
     (gogoproto.nullable) = false
   ];
+  // Login is the host login the user is requesting access for.
+  string Login = 5 [(gogoproto.jsontag) = "login,omitempty"];
+
+  // RequestableResourceIDs is the list of the ResourceIDs of the resources that all
+  // returned requestable roles must allow access to.
+  repeated ResourceID RequestableResourceIDs = 6 [
```

Comment:
I think this field name is pretty unclear, when we already have a `ResourceIDs` field with the same time and a very similar name, I can't tell the intended usage from the name or the comment TBH. I guess using `RequestableResourceIDs` instead of `ResourceIDs` is essentially a flag stating that we want to look for relevant roles for a Role Request instead of a Resource Access Request.

I'd suggest either a name that makes this more clear, or reusing the existing ResourceIDs field and adding a RequestMode enum or something like that